locale/libbout.pot

```diff
@@ -0,0 +1,568 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
Super minor, but this is still all dummy text. I think we can set these through options to `xgettext`, or possibly leave it off all together?

Reply:
I'm not sure, but I think this part gets filled in when creating a .po file for a particular language. `msginit` asks for email addresses and seems to find the package information.